tests/test_healpix_ffts.py

```diff
@@ -1,48 +1,86 @@
 import numpy as np
+from numpy.testing import assert_allclose
 import healpy as hp
 import pytest
 import jax
+if jax.__version__ < "0.4.32":
```

Comment:
String comparison on version specifiers can give incorrect results - [for example](https://stackoverflow.com/q/11887762)

```
>>> "2.3.1" > "10.1.1"
True
```

To be more robust here we would need to something like:

```Python
from packaging.version import Version as _Version

...

if _Version(jax.__version__) < _Version("0.4.32"):
```

and add `packaging` to project dependencies.